### PR TITLE
[13.0][MIG] product_packaging_type_pallet

### PR DESCRIPTION
**Pull request by grindtildeath**
Forward-port of #532 

Depends on:
 - [x] #551 
 - [x] #536

### CODE REVIEW

File: product_packaging_type_pallet/models/product_packaging.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Camptocamp SA
```
Comment (Tonow-c2c) [inline product_packaging_type_pallet/models/product_packaging.py:1]:
```suggestion
# Copyright 2020 Camptocamp SA
```

File: product_packaging_type_pallet/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Camptocamp SA
```
Comment (Tonow-c2c) [inline product_packaging_type_pallet/__manifest__.py:1]:
```suggestion
# Copyright 2020 Camptocamp SA
```

**Review by Tonow-c2c — APPROVED** [review]

**Comment by grindtildeath** [comment]
@Tonow-c2c No need to update the date if the code was written last year :wink:

**Comment by simahawk** [comment]
@grindtildeath rebase and squash needed

**Review by simahawk — APPROVED** [review]

**Pull request merged**